src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private final double radius;
+
+    public Circle(String color, double radius) {
```

Comment:
It is recommended to avoid passing expressions directly into the constructor for better readability. Consider creating a variable for the 'radius' parameter before passing it to the constructor.